utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -498,7 +498,6 @@ Order the given JSON object alphabetically. Would result into:
 
 @{
     name: 'carml'
-    diagnosticLogsRetentionInDays: 7
```

Comment:
Need to update the `.EXAMPLE` to remove the `diagnosticLogsRetentionInDays` examples.

Reply:
removed